.github/workflows/coq-action.yml

```
@@ -0,0 +1,54 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    # the OS must be GNU/Linux to be able to use the docker-coq-action
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          - 'coqorg/coq:dev'
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - uses: coq-community/docker-coq-action@v1
+        with:
+          opam_file: 'coq-paramcoq.opam'
+          custom_image: ${{ matrix.image }}
+          custom_script: |
+            startGroup Print opam config
+              opam config list; opam repo list; opam list
+            endGroup
+            startGroup Build dependencies
+              opam pin add -n -y -k path $PACKAGE $WORKDIR
+              opam update -y
+              opam install -y -j 2 $PACKAGE --deps-only
+            endGroup
+            startGroup Build
+              opam install -y -v -j 2 $PACKAGE
+              opam list
+            endGroup
+            startGroup Workaround permission issue
+              sudo chown -R coq:coq .
+            endGroup
+            startGroup Run tests
+              make -C test-suite
```

according to the old travis file, this should be
```suggestion
              make -C test-suite examples
```